src/components/chat-connect/chat-connect.tsx

```diff
@@ -75,6 +76,17 @@ export class Container extends React.Component<Properties> {
     this.chat = props.chat || chat;
   }
 
+  reconnectStart = () => {
```

Comment:
I think we should move these all up into sagas. I'm not sure why we have a component managing these external connections. But, that's a problem for another day. :)